assets/app/views/_project-page.html

```diff
@@ -16,10 +16,8 @@
     <div class="container-fluid">
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <div class="actions pull-right">
-          <!-- If we have any header actions for the sidebar in the future, they would go here
-              and be links like below
-              <a href="#">Example</a>
-          -->
+	  <a href="#" ng-click="overviewToggle()" ng-if="overviewMode == 'tiles'">Topology</a>
```

Comment:
So I guess use a button group?

https://www.patternfly.org/widgets/#button-groups

Which icons should I use? fa-table and fa-sitemap? Hmmm, although Cockpit uses fa-sitemap for the topology view, Openshift uses it for 'Details'. Any ideas?

http://fortawesome.github.io/Font-Awesome/icon/table/
http://fortawesome.github.io/Font-Awesome/icon/sitemap/